src/main/java/Deadline.java

```diff
@@ -0,0 +1,37 @@
+import java.time.LocalDateTime;
+
+/** Class Deadline that represent a task with a deadline **/
+public class Deadline extends Task{
+    /** Symbol to represent a deadline task **/
+    private static final char SYMBOL = 'D';
+    /** deadline of the task **/
+    private LocalDateTime deadline;
+
+    /**
+     * Constructor of a Deadline task.
```

Comment:
2 points:
1. There should be an "Empty line between description and parameter section" - CS2103 Java Coding Standard.
2. There should be "Punctuation behind each parameter description". I.e. Lines 12 and 13 of Deadline.java should end with a period, as shown below.

Consider the following Javadoc:
```
/**
* Constructor of a Deadline task.
* 
* @param desc Description of the deadline task.
* @param deadline Deadline of the deadline task.
*/
```